plugins/destination/mssql/CONTRIBUTING.md

```diff
@@ -10,6 +10,34 @@ go run main.go serve
 
 ## Testing
 
+Tests require an SQL Server instance available.
+
+You can run the following script to start the local server & create the required database:
+
+```bash
+docker run \
+  --platform=linux/amd64 \
+  -e ACCEPT_EULA=Y \
+  -e MSSQL_PID=Express \
+  -e MSSQL_SA_PASSWORD='yourStrongP@ssword' \
+  -e DB_USER=SA \
+  -e DB_NAME=cloudquery \
+  -p 1433:1433 \
+  -d mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
I tried this on MacOS, but got the following in the logs:

```
SQL Server 2017 will run as non-root by default.
This container is running as user root.
To learn more visit https://go.microsoft.com/fwlink/?linkid=2099216.
/opt/mssql/bin/sqlservr: Invalid mapping of address 0x40094e5000 in reserved address space below 0x400000000000. Possible causes:
1) the process (itself, or via a wrapper) starts-up its own running environment sets the stack size limit to unlimited via syscall setrlimit(2);
2) the process (itself, or via a wrapper) adjusts its own execution domain and flag the system its legacy personality via syscall personality(2);
3) sysadmin deliberately sets the system to run on legacy VA layout mode by adjusting a sysctl knob vm.legacy_va_layout.
```

Is it expected to work on Mac? Anything I'm missing?

Reply:
Possibly Rosetta has to be enabled for Docker?